app/Http/Controllers/Account/AcceptanceController.php

```diff
@@ -347,6 +347,7 @@ public function store(Request $request, $id) : RedirectResponse
 
             $acceptance->decline($sig_filename, $request->input('note'));
             $acceptance->notify(new AcceptanceAssetDeclinedNotification($data));
+            Log::debug('New event acceptance.');
```

Comment:
I don't think this is really needed here

Reply:
ok, i will remove it, BUT before remove it, i try to explain and you just said if are valid or not:

when I decline a consumable device, you've confirmed that a 500 error occurs if the mail system is not already configured, this is already known, but there's no clear reason for it, i've already reviewed the issues and noticed that they don't handle the error from the response mail system.

these lines are more of a marker to identify what was happening before this problem occurred.

sop these lines will help others to find a way to solve this problem, currently i am made testing but i need this previously there for!

Reply:
@snipe  also i see more lines like this in the LDAP part code! Soi i just followed the same practices.